e2e/email-settings.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from '@playwright/test';
+
+const settingsPageElement = {
+  emailSettingsSectionHeader: 'email-settings-header',
+  emailVerificationAlert: 'email-verification-alert',
+  emailVerificationLink: 'email-verification-link',
+  currentEmailText: 'current-email',
+  newEmailInput: 'new-email-input',
+  confirmEmailInput: 'confirm-email-input',
+  saveButton: 'save-email-button',
+  emailSubscriptionYesPleaseButton: 'yes-please-button',
+  emailSubscriptionNoThanksButton: 'no-thanks-button',
+  flashMessageAlert: 'flash-message'
+} as const;
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/settings');
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test.describe('Email Settings', () => {
+  test('should display email settings section header on settings page', async ({
+    page
+  }) => {
+    await expect(
+      page.getByTestId(settingsPageElement.emailSettingsSectionHeader)
+    ).toHaveText('Email Settings');
+  });
+
+  test('should display current email address', async ({ page }) => {
```

Comment:
```suggestion
  test('should display current email address', async () => {
```

Try and read from the global copy of the page in the other tests as well. This might remove the flaky timeouts